benchmarks/cwe_mitre.toml

```diff
@@ -0,0 +1,25 @@
+[general]
+    # benchmark name
+    name = 'CWE'
+    # select the most applicable category from the list provided by this repo
+    category = 'CODING_STANDARD'
+    # mark your benchmark with custom tags (your own custom tags OR use existing from other projects)
+    tags = ['portability', 'security', 'safety']
+
+[info]
+    # programming language
+    language = 'language independent'
```

Comment:
It's usually called `language agnostic`, I think

Reply:
Ok. Thank you